### PR TITLE
Added optional --output parameter + build availability.

### PR DESCRIPTION
**Pull request by andipavllo**

### CODE REVIEW

File: src/remote.js [hunk 1]
```diff
@@ -200,58 +208,69 @@
...
+                      .then(
+                        function(sessionUrl) {
+                          var buildPath = "";
+                          monaca.download(sessionUrl, {}, function(response) {
+                            var filename = "";
+                            if (params.output) {
+                              buildPath = params.output;
+                            } else {
+                              if (typeof response.headers['content-disposition'] === 'string') {
+                                filename = response.headers['content-disposition'].match(/filename="?([^"]+)"?/)[1];
```
Comment (andipavllo) [inline src/remote.js:256]: Fran has already checked if `content-disposition header` matches the regular expression before picking up the matching string. He will implement this.

File: src/remote.js [hunk 2]
```diff
@@ -259,20 +278,20 @@
...
-          },
-          function(err) {
-            util.err("Unable to build this project " + err);
           }
-        )
-      },
-      function(err) {
-        util.err(err);
+        ),
+        function(err) {
```
Comment (andipavllo) [inline src/remote.js:292]: Here I could not force an error, maybe Fran can check what's the type of err object and confirm the user gets an appropriate error message.

File: src/remote.js [hunk 3]
```diff
@@ -259,20 +278,20 @@
...
                     }
                   )
-                },
-                function(error) {
-                  util.err(error);
                 }
-              )
+              },
+              function(error) {
+                util.err("Unable to build this project. " + error);
```
Comment (andipavllo) [inline src/remote.js:284]: now it always returns an `Error` type, I will leave to Fran the change from `error` to `error.message` as he may already have implemented it.

**Comment by masahirotanaka** [comment]
Thanks. Now I'll ask @frankdiox to do some more improvements :)

**Pull request merged**